### PR TITLE
Add reset button to top bar so it looks decent

### PR DESCRIPTION
**Pull request by palmfjord**
also easier than i thought 🤷‍♀️

### CODE REVIEW

File: src/components/TopBar.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const Bar = styled('div')({
   display: 'flex',
   flexDirection: 'row',
   alignItems: 'center',
-  justifyContent: 'end',
+  justifyContent: 'space-between',
```
Comment (frapperino) [inline src/components/TopBar.tsx]: I think this change is the reason the content is skewed to the right when a button isn't present

File: src/pages/Chat/ResetButton.tsx [hunk 2]
```diff
@@ -1,15 +1,31 @@
-import { TranslationsConsumer } from '@hedviginsurance/textkeyfy'
 import * as React from 'react'
+import styled from 'react-emotion'
 import { ChatContainer } from './state'
 
+const Wrapper = styled('div')({
+  paddingRight: 24,
```
Comment (frapperino) [inline src/pages/Chat/ResetButton.tsx]: I would like this to be marginRight: 26 to be consistent with the other buttons:)

Comment (frapperino) [reply src/pages/Chat/ResetButton.tsx]: and a width of 20%, breaking at 850px to 33%, and at 600px to 50%

Comment (palmfjord) [reply src/pages/Chat/ResetButton.tsx]: first comment = 👍
2nd i don't understand. this is what it looks like;
screenshot 2018-10-25 at 13 52 52

Comment (frapperino) [reply src/pages/Chat/ResetButton.tsx]: Ah, sry not so clear comment. Those are the breakpoints for when the progress steps gets replaced by a single step, and lastly not showing progress step at all for smallest size

Comment (palmfjord) [reply src/pages/Chat/ResetButton.tsx]: but why do we need any width in the first place?

**Pull request merged**